frb_dart/lib/src/env/web.dart

```diff
@@ -0,0 +1,10 @@
+// see https://github.com/flutter/flutter/issues/55870
+
+const isWeb = bool.fromEnvironment('dart.library.html');
+final int? portEnv =
+    int.fromEnvironment('PORT') != 0 ? int.fromEnvironment('PORT') : null;
```

Comment:
Btw should this have `const int.fromEnvironment` not `int.fromEnvironement` given that issue?

Reply:
good spot!

Reply:
so here the dart linter prevents the `bool.fromEnvironment` to be `const` for some reason, otherwise done in 59027f2